### ISSUE TITLE
Not able to delete a tweet 

### ISSUE THREAD

**Issue by outterspacem** [oliver-gomes/twitter-clone-yt#1]
Hi all,

I am reaching the end of part 1 of the tutorial to build the backend. 

I am able to create a tweet successfully but I am not able to delete a tweet.

Here is the code for the "Tweet.js" file in my controllers folder:

` import Tweet from '../models/Tweet.js';
import { handleError } from '../error.js';
import User from "../models/User.js";

export const createTweet = async (req, res, next) => {
    const newTweet = new Tweet(req.body);
    try {
        const savedTweet = await newTweet.save();
        res.status(200).json(savedTweet);
    } catch (error) {
        handleError(500, error); 
    }
};

export const deleteTweet = async (req, res, next) => {
  try {
    const tweet = await Tweet.findById(req.params.id);
    if (tweet.userId === req.body.id) {
      await tweet.deleteOne();
      res.status(200).json("tweet has been deleted");
    } else {
      handleError(500, err);
    }
  } catch (err) {
    handleError(500, err);
  }
}; `

What could the issue be? 

Thanks

**Comment by oliver-gomes** [comment]
@outterspacem can you double check if you are verifying the user before deleting 

Also how are you testing this in postman

There's no error in controller so it's coming from outside of it

![del](https://user-images.githubusercontent.com/32399333/213946537-d47e7ed2-8609-44ef-b77a-46013dc572ad.png)

**Comment by outterspacem** [comment]
@oliver-gomes Yes, for example when I try to delete a tweet from "harryp", I'm making sure I am first logged in as "harryp" and passing on the right token. I am following the same steps on the video on Postman in order to test. I even tried Thunder Client on VS Code (similar to Postman) but it's still not working so I don't think the issue is Postman. I also double checked my "tweets.js" file in the routers folder and the route is properly defined. I finished part 1 of the tutorial and everything works fine except deleting a tweet for some reason.
